modules/nodes-nodes-working-deleting.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-managing.adoc
+
+[id='nodes-nodes-managing-deleting_{context}']
+= Deleting nodes from a cluster
+
+When you delete a node using the CLI, the node object is deleted in Kubernetes,
+but the pods that exist on the node are not deleted. Any bare pods not
+backed by a replication controller become inaccessible to {product-title}.
+Pods backed by replication controllers are rescheduled to other available
+nodes. You need to delete local manifest pods.
+
+.Procedure
+
+To delete a node from the {product-title} cluster:
+
+. Evacuate pods from the node you are preparing to delete.
```

Comment:
add the `oc adm drain` example command here

Reply:
i see there is an evacuation section below.  it should come first.

Reply:
Added:
```
. Evacuate pods from the node you are preparing to delete.
+
----
$ oc adm drain <node> [--pod-selector=<pod_selector>]
----

```